.travis.yml

```diff
@@ -1,16 +1,21 @@
-sudo: false
+sudo: required # due to manual install of make 4.1
```

Comment:
That's what I thought, yeah. Then I checked the log at https://travis-ci.org/nodemcu/nodemcu-firmware/builds/267828141 and expanded "Build system information". Since it's got Docker version  info there I was assuming that it'd be running on Docker now despite sudo.